Library/Homebrew/exceptions.rb

```diff
@@ -65,6 +65,21 @@ def to_s; <<-EOS.undent
   end
 end
 
+class TapFormulaAmbiguityError < RuntimeError
+  attr_reader :name, :paths
+
+  def initialize name, paths
+    @name = name
+    @paths = paths
+
+    super <<-EOS.undent
+      Multiple formulae found: #{paths.map { |f| "\n       * #{f}" }.join}
+
+      Please use the fully-qualified name to refer the formula.
```

Comment:
Sorry, I mean `Please use the fully-qualified name e.g. user/tap/formula to refer the formula.`